src/Generators/FactoryGenerator.php

```diff
@@ -83,6 +84,17 @@ protected function buildDefinition(Model $model)
                 if (in_array($column->dataType(), ['enum', 'set']) and !empty($column->attributes())) {
                     $definition = str_replace("/** {$column->dataType()}_attributes **/", json_encode($column->attributes()), $definition);
                 }
+
+                if (in_array($column->dataType(), ['float', 'decimal']) and !empty($column->attributes())) {
+                    $precision = intval(Arr::last($column->attributes()));
+                    $scale = intval(Arr::first($column->attributes()));
+
+                    $definition = str_replace(
+                        "/** {$column->dataType()}_attributes **/",
+                        implode(', ', [$precision, 0, intval(str_repeat(9, $scale)) / pow(10, $precision)]),
```

Comment:
This math is slightly off. `scale` is _max digits_. But really it should be _max digits_ - `$precision`. [Reference](https://dev.mysql.com/doc/refman/5.7/en/precision-math-decimal-characteristics.html)